kolibri/plugins/management/templates/management/management.html

```diff
@@ -1,4 +1,19 @@
-{% extends "kolibri/base.html" %}
-{% block content %}
-<span>The management app!</span>
-{% endblock %}
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kolibri</title>
+  </head>
+  <body>
+    <main v-ref:main></main>
```

Comment:
I'm having trouble figuring out what `v-ref:main` is used for. I see similar code in `test/management.js`